ui/webpack.prod.js

```diff
@@ -0,0 +1,24 @@
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+
+const common = require('./webpack.common.js');
+
+/**
```

Comment:
I believe comment in style /** ... */ is for JSDoc (equivalent of JavaDoc):
http://usejsdoc.org/about-getting-started.html

Normal comment like this should use // or /* ... */. If you mean this to be a JSDoc, add description of this constant as well.

Reply:
Done.